Packages/com.unity.inputsystem/Documentation~/PreReleaseNotes.md

```diff
@@ -2,91 +2,33 @@
 
 ## Overview
 
-This pre-release contains updates to the Input System which simplifies and improves some of the main workflows compared with earlier versions of the Input System package.
+This pre-release contains updates to the Input System which simplify and improve some of the main workflows compared with earlier versions of the Input System package. This page describes the main differences introduced, and assumes you are familiar with the workflow in the previous versions of the Input System package.
 
-Because this is a pre-release, the rest of the documentation included with this version of the package has not yet been updated to reflect these changes. The documentation on this page explains the improvements and differences between these new features, and the previous version of the input system package. The improvements are as follows:
+## New project-wide actions
 
-**New project-wide actions**
-
-The Input System now allows you to configure actions in the Project Settings window, in a new Input Actions Settings panel. The actions configured here apply project-wide. This means you no longer need to create an Actions asset and set up a reference to your asset to read input actions. Instead, you can configure your actions in the Project Settings window, and read them directly from your scripts. You can still use Action assets if you like, but for many typical scenarios, they are no longer necessary.
-
-**New default actions**
-
-The new project-wide actions come pre-loaded with some default action maps that contain actions suitable for many typical game scenarios, including basic player character controls, and typical UI interactions. In many cases these are enough to allow you to immediately start using input in your project with no configuration required. You are free to either add to, edit, or delete these default configurations to suit your needs.
-
-
-**Note**: The new features in this pre-release are **only documented on this page**. When reading the rest of the documentation in this package, please remember that it *will not mention these features*. So, for example, when another page in the documentation discusses **action assets**, or creating a reference to your action asset, you can in many cases use project-wide actions instead, as described on this page.
-
-## Project-wide actions and default actions
+The Input System now allows you to configure actions in the Project Settings window, in a new Input Actions Settings panel. The actions configured here apply project-wide. This means you no longer need to create an Actions asset and set up a reference to your asset to read input actions. Instead, you can configure actions in the Project Settings window, and read them directly from your scripts. You can still use Action assets if you like, but for many typical scenarios, they are no longer necessary.
```

Comment:
"The actions configured here apply project-wide", would it make more sense to say "The actions configured here are available anywhere in the project"